src/components/jellyfinApi.ts

```diff
@@ -75,9 +71,39 @@ export abstract class JellyfinApi {
     }
 
     // Authenticated ajax
-    public static authAjax(url: string, args: any): Promise<any> {
+    public static authAjax(path: string, args: any): Promise<any> {
```

Comment:
```suggestion
    public static authAjax(path: string, args: any): Promise<void> {
```

Same as above with all the ``any``'s

Reply:
We use the result, sorry